packages/expo-barcode-scanner/src/ExpoBarCodeScannerView.ts

```diff
@@ -2,6 +2,6 @@ import { requireNativeViewManager } from 'expo-modules-core';
 import React from 'react';
 
 const ExpoBarCodeScannerView: React.ComponentType<any> =
-  requireNativeViewManager('ExpoBarCodeScannerView');
+  requireNativeViewManager('ExpoBarCodeScanner');
```

Comment:
Yeah, I was going to do the android side next but I can change it now if you think that'd be better?

Reply:
Will just changing the name work? Just wondering because, with the old API, they are named separately and now it's the same string for both imports.

Reply:
Might be better to revert the name for this PR and change it when Android is done

Reply:
If you're going to work on the Android side I think that's fine to change it there and leave it as is here. Since we're holding on with merging this PR, I guess we can merge both at the same time 🙂